stable/ambassador/templates/deployment.yaml

```diff
@@ -89,6 +95,9 @@ spec:
             - name: stats-exporter-mapping-config
               mountPath: /statsd-exporter/
               readOnly: true
+            - name: ambassador-pod-info
+              mountPath: /tmp/ambassador-pod-info
+              readOnly: true
```

Comment:
This is added to the wrong container, it should be line 173 or 164 before this diff ( https://github.com/helm/charts/blob/9522acbfd161903590a03c032803b873a95775ab/stable/ambassador/templates/deployment.yaml#L164 )

Reply:
Good catch, thanks ! I've moved it into the right container.